Automattic/jetpack
# Protect: Add scanning spinner to plugin admin

Description #
- This PR adds a spinner to the Protect plugin page in `wp-admin` when a scan is in progress.

#### Changes proposed in this Pull Request:

* The change applies a [core/Gutenberg spinner component](https://wordpress.github.io/gutenberg/?path=/docs/components-spinner--default) to the section that displays upon the initial setup of the Protect plugin and/or during the regeneration of a status report.
* The structure of this conditional display is reorganized to account for the additional content and to supply adequate spacing for each element for various viewports and devices.

#### Other information:

#### Jetpack product discussion


- 1201069996155224-as-1202476651392832

#### Does this pull request change what data or activity we track or use?


- No

#### Testing instructions:






* Create a [Jurassic Ninja](https://jurassic.ninja/) testing site w/ Jetpack installed
* When the instance is ready, log in and go to Plugins and install the [Jetpack Beta plugin](https://jetpack.com/download-jetpack-beta/)
* Once the beta plugin is activated, search for and activate the current Feature Branch of Jetpack Protect from the Jetpack Beta plugin page link in `wp-admin`
* Visit Jetpack > Protect and allow your initial status report to be generated
* Verify that at the top of the in-progress scan display there is an animated spinner accompanied by the following text: `Scanning your site...`
* Ensure that once the report is complete, you're appropriately passed on to the scan summary display and that the spinner and text are no longer present
